File: SportsStore.Web/ClientApp/components/models/Product.ts

export class Product {

	public id: number = 0;
	public description: string = "";
	public category: string = "";
	public price: number = 0;
	public name:string="";
}

export interface IProduct {
	id: number;
	description: string;
	category: string;
	price: number;
	name: string;
}

File: SportsStore.Api/Models/Entities/Product.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace SportsStore.Api.Models.Entities
{
    public class Product : BaseEntity
    {
        [Required]
        public string Name { get; set; }
        [Required]
        public string Description { get; set; }
        [Required]
        public decimal Price { get; set; }
        [Required]
        public string Category { get; set; }
    }
}
